BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -91,6 +91,15 @@ public final class BlueprintView: UIView {
         }
     }
 
+    /// We need to invalidateIntrinsicContentSize when `bound.size` changes for Auto Layout to work correctly.
+    public override var bounds: CGRect {
```

Comment:
Double thinking through this: We used to `invalidateIntrinsicContentSize` in `layoutSubviews`, is there any benefit or drawback to doing it here instead? This seems better since we now only invalidate when our bounds as applied by autolayout actually change, but the timing is slightly different from `layoutSubviews`.

Reply:
Yeah I was wondering about that myself, but if this works it seems better to me since it'll be hit less?